+++ ./opt/python3.10/lib/python3.10/site-packages/repoman/repomanager.py	2023-07-26 11:46:25.868183171 +0200
@@ -2,7 +2,6 @@
 import os,sys,shutil
 import json
 import subprocess
-from appconfig import appConfigN4d
 import requests
 import subprocess
 from requests.adapters import HTTPAdapter
@@ -772,15 +771,7 @@
 	#def reversePinning
 
 	def isMirrorEnabled(self):
-		sw=True
-		n4d=appConfigN4d.appConfigN4d()
-		ret=n4d.n4dQuery("MirrorManager","is_mirror_available")
-		if isinstance(ret,dict):
-			if str(ret.get("status","-1"))!="0":
-				sw=False
-		elif isinstance(ret,str):
-			if ret!="Mirror available":
-				sw=False
+		sw=False
 		return(sw)
 	#def isMirrorEnabled
 	
